man/adoc/bpftrace.adoc

```diff
@@ -1789,8 +1789,8 @@ You can also pass the address type (e.g. AF_INET) explicitly as the first parame
 === offsetof
 
 .variants
-* `offsetof(STRUCT, FIELD)`
-* `offsetof(EXPRESSION, FIELD)`
+* `offsetof(STRUCT, FIELD[.SUBFIELD1.SUBFIELD2])`
+* `offsetof(EXPRESSION, FIELD[.SUBFIELD1.SUBFIELD2])`
```

Comment:
```suggestion
* `offsetof(STRUCT, FIELD[.SUBFIELD])`
* `offsetof(EXPRESSION, FIELD[.SUBFIELD])`
```

I would also add example and a small comment that we support any number of sub field levels.

Reply:
Thanks, just update, please review.